docker-compose.yml

```diff
@@ -68,12 +68,12 @@ services:
     depends_on:
      - db-configure
     volumes:
-     - "${DOCKER_MW_PATH}:/var/www/mediawiki"
+     - "${DOCKER_MW_PATH}:/var/www/mediawiki:cached"
      - ./config/mediawiki:/var/www/mediawiki/.docker:ro
      - ./config/hhvm/php.ini:/etc/hhvm/php.ini:ro
      - ./config/hhvm/server.ini:/etc/hhvm/server.ini:ro
      - ./scripts/wait-for-it.sh:/srv/wait-for-it.sh:ro
-     - mw-images:/var/www/mediawiki/images/docker
+     - mw-images:/var/www/mediawiki/images/docker:delegated
```

Comment:
The `delegated` setting has the least guarantees about how fast changes from the appear on the guest. Given this should almost never be written to by the host (and is expected to be written to in the container), makes sense to be conservative here.

Note that we're only talking about a tens or hundreds of milliseconds at most. Nothing a refresh or retry won't fix in the worst case scenario. By default, though, Docker tries to make it atomic and perfect in sync, hence the performance issues.

For MW code, I've settled on `cached` instead of `delegated` which is slightly favouring writes from the host to appear fast on the container, given MW should mostly not write to that directory aside from caching and maintenance scripts. I did that because I was not able to find a significant/consistent improvement from `cached` to `delegated` so being conservative towards status quo.